### PR TITLE
Better behavior for invalid records.

### PR DESCRIPTION
**Pull request by CuriousG102**
@rdmurphy I'm trying to test it with your Tamu data but my computer isn't cooperating. This can be expected to create a merge conflict with my other modification to generic_merge_cell_transform when you eventually merge that into master.

### CODE REVIEW

**Comment by CuriousG102** [comment]
@rdmurphy It works.

**Comment by CuriousG102** [comment]
Will make that change. Sorry I just now saw your message.

**Comment by CuriousG102** [comment]
Bump?

**Comment by CuriousG102** [comment]
Modified to print all warnings

File: tx_salaries/utils/transformers/base.py [hunk 1]
```diff
@@ -115,11 +117,18 @@ def generic_transform(labels, source, record_class):
...
     data = []
+    warnings = []
     for raw_record in source:
         record = record_class(dict(zip(labels, raw_record)))
         if record.is_valid:
             data.append(record.as_dict())
-    return data
+        else:
+            warnings.append("WARNING: RECORD INVALID; {0}".format(record.data))
+    if len(warnings) != 0 and record_class.REJECT_ALL_IF_INVALID_RECORD_EXISTS:
```
Comment (rdmurphy) [inline tx_salaries/utils/transformers/base.py]: Could this just be `if warnings...`? The empty list would be false-y.

Comment (CuriousG102) [inline tx_salaries/utils/transformers/base.py]: Well, the PEP8 gods apparently made a commandment, so I better make it so
https://www.python.org/dev/peps/pep-0008/#programming-recommendations

**Comment by CuriousG102** [comment]
@rdmurphy code has been updated to be more stylish

**Comment by rdmurphy** [comment]
Thanks! Otherwise this was cool. I'll pull it in. :+1:

**Pull request merged**